Content: 2 new jobs kinds for the Unfettered, mining and medical rush (#7444)
endless-sky/endless-sky: data/hai/unfettered jobs.txt

		`A group of Unfettered unload the materials you mined, without saying anything to you. When they are done, one gives you <payment>.`
		`The leader of the Unfettered unloading the materials comes to you. "This will allow us to repair our houses, our hospitals, even our schools. For that you have my deepest thanks." He then hands you <payment>.`
		`The local Unfettered collect the minerals and happily hand you <payment>.`
		`Once the unloading of materials is finished, an Unfettered comes to you, looking humble and sincere. "Thank you, captain. We need these materials - and any other kind of help - more than we would like to admit it. If you have time, please consider doing other jobs, such as bringing food for us," he says with a slight smile. He then gives you <payment>.`

mission "Unfettered Silicon Asteroid Mining"
	job
	repeat
	name `Gather silicon`
	description `Mine and then collect 50 tons of silicon from the asteroid field in Wah Yoot, and then bring them back to <destination> for <payment>.`
	source "Darkcloak"
	to offer

> 		random < 40

Author: They could bring in multiple small parts or smth, or do it later
And 40 tons isnt much

> 		random < 40

Author: Yeah it is this is just basic materials that they buy back from you, as if they were buying food
Tributes require more trust than this

> 		random < 40

Author: Maybe the medicals could require a bit more... I imagine they only ask you if they really have to

> 		random < 40

Reviewer: 50 tons is a lot for someone in a single ship, and even more so if they're using outfit expansions. (Knowing to put minerals in storage is also pretty unintuitive.)

> 		random < 40

Author: I'll do it for the big mission but I want the player to see these even if they dont have cargo rn

> 		random < 40

Author: I made it for all missions, if you have a bit under it you'll see them and the smallest one is always offered regardless of cargo so the player knows its smth they can do

> 		random < 40

Author: I made them be more likely to offer if you've done the JD
But if they really need to they'll ask you anyway

> 		random < 40

Author: I hid the big one and made the small one more unlikely, should be good now I think

	job
	repeat
	name `Express medical supplies to <planet>`
	description `The Unfettered hospitals on Darkcloak are in urgent need of resupply. Collect <tons> of medical supplies from <stopovers>, and bring it to the Unfettered on <destination> by <date> for <payment>.`
	deadline 10 1
	cargo "medicals" 15 2 .05
	source
		attributes "unfettered"
	stopover
		distance 5 7
		government "Hai"
		attributes "spaceport"
	destination "Darkcloak"
	to offer
		has "Unfettered Jump Drive 1: offered"
		random < 60
	on visit
		dialog phrase "generic missing stopover or cargo"
	on stopover
		dialog phrase "unfettered medicals pickup dialog"
	on complete
		payment 10000 4000
		dialog phrase "unfettered medicals payment dialog"

mission "Unfettered Big Medicals Rush"
	job
	repeat
	name `Medical supplies to <planet>`
	description `The Unfettered hospitals on <destination> are in need of resupply. Collect <tons> of medical supplies from <stopovers>, and bring it to the Unfettered on <destination> by <date> for <payment>.`
	deadline 15
	cargo "medicals" 35 2 .05
	source
		attributes "unfettered"
	stopover
		distance 5 7
		government "Hai"
		attributes "spaceport"
	destination

> 		attributes "unfettered"

Author: I dont like that, it makes it messy, all off them pointing to the same world
And the world would call for this on the nearby ones

> 		attributes "unfettered"

Author: Also all planets need food, but not all will need meds all the time

> 		attributes "unfettered"

Reviewer: The vast majority of stopover jobs point back at the source; I don't see a strong argument for not doing that for this mission.

> 		attributes "unfettered"

Author: did it that way now, I'll do all it takes for it to be merged x)

	to offer

> 		has "Unfettered: Bring Medical Supplies: done"

Reviewer: Is this a real mission?

Author: Oh crap not meant to be there

		random < 40
	on visit
		dialog phrase "generic missing stopover or cargo"
	on stopover
		dialog phrase "unfettered medicals pickup dialog"
	on complete

> 		payment 8000 3000

Author: Oh I did not mean to make the deadline stricter but the opposite

> 		payment 8000 3000

Reviewer: Better, but the payment for both rush missions still feels too high. Doing a little bit of testing, I've found jobs that pay 5.6 million for 101 tons, 2.7 million for 69 tons and 2.8 million for 59 tons.

> 		payment 8000 3000

Reviewer: The missions are pretty lenient, though. On the maximum distance job, you're getting ~24 days to travel 14 jumps.

> 		payment 8000 3000

Author: I'm kinda new with these kind of jobs tbh, I made some tweaks to the payments and how likely they are to offer
 I'll make the first mission a bit more stressful then

> 		payment 8000 3000

Author: It doesnt count the way back actually, so I'll modify it a bit.
(the second thing in deadline is only the distance until the stopover and without taking the way back into account)

> 		payment 8000 3000

Author: Nvm I was wrong on that, changed it hope its good now

		outfit "Hyperdrive" 1
		payment 1000000
		"reputation: Hai (Unfettered)" >?= 40

phrase "unfettered mining payment dialog"
	word
		`The group of Unfettered unloading the materials you mined seems enthused. "Keep this up and we will be building new ships in no time! Some use asteroids as target practice, but find it too dull and easy to fight an enemy that does not react," one tells you while handing over your payment of <payment>.`

> 		`The local Unfettered unload your cargo of minerals, and one comes you give you your payment of <payment>. "Some Unfettered use these asteroids as target practices, but too few bring the precious materials back," he adds.`

Reviewer 2: ```suggestion
		`The local Unfettered unload your cargo of minerals, and one comes to give you your payment of <payment>. "Some Unfettered use these asteroids as target practices, but too few bring the precious materials back," he adds.`
```

		`The Unfettered unloading the materials you mined seem satisfied. "This will allow us to repair our ships! Too few captains are patient enough to gather asteroids like you are. Most of them try to get the resources from enemy ships, but this is a safer way to do it," one tells you before doling out your payment of <payment>.`
		`A group of Unfettered unload the materials you mined, without saying anything to you. When they are done, one gives you <payment>.`
		`The leader of the Unfettered unloading the materials comes to you. "This will allow us to repair our houses, our hospitals, even our schools. For that you have my deepest thanks." He then hands you <payment>.`
		`The local Unfettered collect the minerals and happily hand you <payment>.`
		`Once the minerals are unloaded, an Unfettered comes to you, looking humble and sincere. "Thank you, Captain. We need these materials - and any other kind of help - more than we would like to admit it." He then gives you <payment>.`

mission "Unfettered Silicon Asteroid Mining"
	job
	repeat
	name "Gather Silicon from asteroids"

> 	description "Mine and collect 50 tons of silicon from the asteroid field in Wah Yoot, then bring the minerals to <destination> for <payment>."

Reviewer 2: Does it matter where you get the minerals from? It seems weird to have the system in the description like it was the only option, but as far as I can see this doesn't actually matter (and is just a recommendation / help).

Author: Yeah its just a recommendation
At first I had set a waypoint but then u cant complete it without going there

		`The local Unfettered unload your cargo of minerals, and one comes to give you your payment of <payment>. "Some Unfettered use these asteroids as target practice, but too few bring the precious materials back," he says.`
		`The Unfettered unloading the materials you mined seem satisfied. "This will allow us to repair our ships! Too few captains are patient enough to gather asteroids like you are. Most of them try to get the resources from enemy ships, but this is a safer way to do it," one tells you before doling out your payment of <payment>.`
		`A group of Unfettered unload the materials you mined, without saying anything to you. When they are done, one gives you <payment>.`
		`The leader of the Unfettered unloading the materials comes to you. "This will allow us to repair our houses, our hospitals, even our schools. For that you have my deepest thanks." He then hands you <payment>.`
		`The local Unfettered collect the minerals and happily hand you <payment>.`

> 		`Once the minerals are unloaded, an Unfettered comes to you, looking humble and sincere. "Thank you, Captain. We need these materials - and any other kind of help - more than we would like to admit it." He then gives you <payment>.`

Reviewer 3: The first and third lines seem to have very similar structures and could probably be split up into words for more variety in the resulting phrases. Same for the fifth and seventh lines.

> 		`Once the minerals are unloaded, an Unfettered comes to you, looking humble and sincere. "Thank you, Captain. We need these materials - and any other kind of help - more than we would like to admit it." He then gives you <payment>.`

Reviewer 3: ```suggestion
		`Once the minerals are unloaded, an Unfettered comes to you, looking humble and sincere. "Thank you, Captain. We need these materials - and any other kind of help - more than we would like to admit." He then gives you <payment>.`
```

> 		`Once the minerals are unloaded, an Unfettered comes to you, looking humble and sincere. "Thank you, Captain. We need these materials - and any other kind of help - more than we would like to admit it." He then gives you <payment>.`

Reviewer 3: ```suggestion
		`Once the minerals are unloaded, an Unfettered comes to you, looking humble and sincere. "Thank you, Captain. We need these materials - and any other kind of help - more than we would like to admit." He then gives you <payment>.`
```

	job
	repeat
	name "Express medical supplies to <planet>"
	description "The Unfettered hospitals on Darkcloak are in urgent need of resupply. Collect <tons> of medical supplies from <stopovers>, and bring it to the Unfettered on <destination> before <date> for <payment>."
	deadline 4 2
	cargo "medical supplies" 15 2 .05
	source
		attributes "unfettered"
	stopover
		distance 5 7
		government "Hai"
		attributes "spaceport"
	destination "Darkcloak"
	to offer
		random < 30 + "Unfettered Jump Drive 1: offered" * 30
	on visit
		dialog phrase "generic missing stopover or cargo"
	on stopover
		dialog phrase "unfettered medicals pickup dialog"
	on complete
		payment 8000 3500
		dialog phrase "unfettered medicals payment dialog"

mission "Unfettered Big Medicals Rush"
	job
	repeat
	name "Medical supplies to <planet>"
	description "The Unfettered hospitals on <destination> are in need of resupply. Collect <tons> of medical supplies from <stopovers>, and bring it to the Unfettered on <destination> before <date> for <payment>."
	deadline 12 2
	cargo "medicals" 35 2 .05
	source
		attributes "unfettered"
	stopover
		distance 5 7
		government "Hai"
		attributes "spaceport"

> 	destination
> 		attributes "unfettered"

Author: alr
```suggestion
```

		outfit "Hyperdrive" 1
		payment 1000000
		"reputation: Hai (Unfettered)" >?= 40

phrase "unfettered mining payment dialog"
	word
		`The group of Unfettered unloading the materials you mined seems enthused. "Keep this up and we will be building new ships in no time! Some use asteroids as target practice, but find it too dull and easy to fight an enemy that does not react," one tells you while handing over your payment of <payment>.`

> 		`The local Unfettered unload your cargo of minerals, and one comes to give you your payment of <payment>. "Some Unfettered use these asteroids as target practices, but too few bring the precious materials back," he adds.`

Reviewer 4: ```suggestion
		`The local Unfettered unload your cargo of minerals, and one comes to give you your payment of <payment>. "Some Unfettered use these asteroids as target practice, but too few bring the precious materials back," he says.`
```

	source
		attributes "unfettered"
		not planet "Warfeed"
	destination "Darkcloak"
	to offer
		random < 40
	on complete
		payment 450000
		dialog phrase "unfettered mining payment dialog"
		outfit "Silver" -15

mission "Unfettered Silver Asteroid Mining 2"
	job
	repeat
	name "Gather Silver from asteroids"
	description "Mine and collect 30 tons of silver, then bring the minerals to <destination> for <payment>. Silver is available in the asteroid field in Ehma Ti."
	source
		attributes "unfettered"
		not planet "Warfeed"
	destination "Darkcloak"
	to offer
		random < 10
		"cargo space" > 25
	on complete
		payment 1000000
		dialog phrase "unfettered mining payment dialog"
		outfit "Silver" -30

mission "Unfettered Lead Asteroid Mining"
	job
	repeat
	name "Gather Lead from asteroids"
	description "Mine and  collect 35 tons of lead, then bring the minerals to <destination> for <payment>. Lead is available in the asteroid field in Wah Yoot."
	source
		attributes "unfettered"
		not planet "Warfeed"
	destination "Darkcloak"
	to offer
		random < 40
		"cargo space" > 30
	on complete

> 		payment 200000

Reviewer 5: ```suggestion
		payment 95000
```

		dialog phrase "unfettered mining payment dialog"
		outfit "Lead" -35

mission "Unfettered Lead Asteroid Mining 2"
	job
	repeat
	name "Gather Lead from asteroids"
	description "Mine and collect 70 tons of lead, then bring the minerals to <destination> for <payment>. Lead is available in the asteroid field in Wah Yoot."
	source
		attributes "unfettered"
		not planet "Warfeed"
	destination "Darkcloak"
	to offer
		random < 10
		"cargo space" > 55
	on complete

> 		payment 450000

Reviewer 5: ```suggestion
		payment 210000
```

		outfit "Hyperdrive" 1
		payment 1000000
		"reputation: Hai (Unfettered)" >?= 40

phrase "unfettered mining payment dialog"
	word
		"The group of"
		"The"
	word
		" Unfettered unloading the materials"
	word
		" you mined"
		" you gathered"
		" you brought"
	word
		" seem "

> 		" appear"

Reviewer 2: ```suggestion
		" appear "
```

	word
		"satisfied."
		"enthused."
		"content."
		"happy."
	word

> 		`"Keep this up and we will be building new ships in no time!`

Reviewer 2: ```suggestion
		` "Keep this up and we will be building new ships in no time!`
```

> 		`"This will allow us to repair our ships!`

Reviewer 2: ```suggestion
		` "This will allow us to repair our ships!`
```

	word
		" one tells you "
	word
		"while handing over"
		"before doling out"
	word
		" your payment of <payment>."

phrase "unfettered mining payment dialog"
	word
		`The local Unfettered unload your cargo of minerals, and one comes to give you your payment of <payment>. "Some Unfettered use these asteroids as target practice, but too few bring the precious materials back," he says.`
		`A group of Unfettered unload the materials you mined, without saying anything to you. When they are done, one gives you <payment>.`
		`The local Unfettered collect the minerals and happily hand you <payment>.`

phrase "unfettered mining payment dialog"
	word
		"The leader of the Unfettered unloading the materials comes to you."
		"Once the minerals are unloaded, an Unfettered comes to you, looking humble and sincere."
	word
		` "`
	word
		`This will allow us to repair our houses, our hospitals, even our schools. For that you have my deepest thanks.`
		`Thank you, Captain. We need these materials - and any other kind of help - more than we would like to admit.`
	word
		`" `
	word

> 		"They then gives you <payment>."

Reviewer 2: ```suggestion
		"They then give you <payment>."
```
